gui/resources/icons/__init__.py

```diff
@@ -0,0 +1,2 @@
+from os.path import abspath, dirname
+icons_dir = abspath(dirname(__file__))
```

Comment:
IMO this belongs more in the `resources`'s `__init__`.

Reply:
have fixed this